staff/YaninaGomezCucchiararo/blog/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Healthy BLOG</title>
+    <link rel="icon" href="images/iconhealthy.png" type="image/png">
+</head>
+
+
+<body>
+    <header>
+        <img src="images/iconhealthy.png" alt="icon healthy" title="Healthy blog">
+        <h1>HEALTHY BLOG</h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="">About us</a>
+                </li>
+                <li>
+                    <a href="">All recipies</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+
+        <section>
+            <h3>LAST RECIPIES</h3>
+            <article>
+                <aside>
+                    <img src="images/lasanya.jpg" title="lasanya image">
+                </aside>
+
+                <h1>LASAÑA DE TOFU CREAM Y CALABACÍN</h1>
+                <p>At home you can never eat pasta, I do not know what it is but it is always well received by our palates and
+                    stomachs hehehe ... So let's try new things experimenting with what there is at home and in an improvisation
+                    morning, this delicious lasagna of tofu-crema with zucchini and tomato emerged. He baptized the title
+                    with the term tofu-crema because I am very imaginative when I want, jurjur ... No, he called it that
+                    because he passed it through the raw processor and he got an unctuous consistency pâté type that has
+                    been super good to bind all the vegetable sauce.
+                </p>
+            </article>
+
+            <article>
+                <aside>
+                    <img src="images/lasanya.jpg" title="lasanya image">
+                </aside>
+
+                <h1>MUFFINS</h1>
```

Comment:
Maybe would be better to use "H2" instead of "H1" because seems more important than the title of the website.